README.md

```diff
@@ -2,52 +2,54 @@
+## On-prem
+- ✨ **BETA** [Docker](/beta/docker)
+- [Docker Compose & Docker Swarm](/docker)
+- [Kubernetes](/kubernetes)
 
-- Azure Container Apps
-  - [CLI Deployment](https://support.1password.com/scim-deploy-azure/) - recommended for most Azure Container App deployments
-  - [Azure Portal Deployment using an ARM Template](/azure-container-apps-arm/README.md) - recommended for those not able to use the tools in the above CLI deployment guide, faster than the manual guide below.
-  - [Azure Portal Deployment (Manual)](/azure-container-apps/README.md) - recommended for those not able to use the tools in the above CLI deployment guide
-  - [Container Apps advanced customizations](/azure-container-apps/ADVANCED.md)
-- DigitalOcean App Platform 
-  - [DigitalOcean Portal Deployment](https://support.1password.com/cs/scim-deploy-digitalocean-ap/)
-  - [DigitalOcean command line tool & the 1Password CLI Deployment](/do-app-platform-op-cli) 
+## Choosing a deployment method
 
-### Advanced deployment options
-If you have particular requirements for your environment, we recommend an advanced deployment. These example configurations will give you a base to create the deployment from, as well as explain what 1Password SCIM Bridge needs to function and how to maintain your bridge once you've deployed it.
+Choosing a deployment method comes down to which infrastructure and tool you mainly use in your organization. Under the hood they all use the same Docker image and serve the same purpose of a SCIM bridge. Below are some more information on terminologies used in each deployment method.
 
-- [AWS ECS Fargate with Terraform](/aws-ecsfargate-terraform)
-- [Azure Kubernetes Service](https://support.1password.com/cs/scim-deploy-azure-kubernetes/)
-- [Docker Compose & Docker Swarm](/docker)
-- [Kubernetes](/kubernetes)
+### Marketplace apps
+For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean.
+
+### Containers as a service (Caas) deployment options
+Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
 
-### Beta deployment
 
-These are beta versions of 1Password SCIM Bridge deployment examples. These deployments _should_ work, but aren't guaranteed and will change in the future. See the [README](./beta/README.md) for more information about the "beta" designation.
+### Beta deployment methods
+The beta deployments _should_ work, but aren't guaranteed and will change in the future. See the [README](./beta/README.md) for more information about the "beta" designation.
 
-- [AWS with CloudFormation](/beta/aws-ecsfargate-cfn)
-- [Docker](/beta/docker)
-- ✨ **NEW** [Google Cloud Run](/beta/google-cloud-run)
 
 ## Deprecated deployment methods
 
```

Comment:
Prefer examples. Suggest we cut out the rest of this section as well (I can't suggest changes to lines that weren't amended).

```suggestion
## Deprecated deployment examples

We sometimes deprecate deployment examples that become irrelevant, usually when a new deployment example is released that is a more desirable solution for that cloud service provider or platform.

### Recently deprecated

 - [Docker Compose & Docker Swarm](https://github.com/1Password/scim-examples/blob/main/deprecated/docker)

See the [README](/deprecated/README.md) in the `/deprecated` directory for more information about this designation and suggested alternatives.

```